lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -791,6 +791,21 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value other = convertScalarToDtype(b, loc, operands[1], dtype);
     return b.create<arith::DivFOp>(loc, self, other);
   }
+  if (auto remScalar = dyn_cast<AtenRemainderScalarOp>(op)) {
+    MLIRContext *context = op->getContext();
+    Type newResultType = converter->convertType(remScalar.getType())
+                             .cast<RankedTensorType>()
+                             .getElementType();
+    Type floatDtype = mlir::FloatType::getF64(context);
+    Value self = convertScalarToDtype(b, loc, payloadArgs[0], floatDtype);
+    Value other = convertScalarToDtype(b, loc, operands[1], floatDtype);
+    Value division = b.create<arith::DivFOp>(loc, self, other);
+    Value floor = b.create<math::FloorOp>(loc, division);
+    Value multiply = b.create<arith::MulFOp>(loc, floor, other);
+    Value subtract = b.create<arith::SubFOp>(loc, self, multiply);
```

Comment:
I think you can use `arith::remf` and `arith::remi` https://mlir.llvm.org/docs/Dialects/ArithmeticOps/#arithremf-mlirarithremfop